Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -211,6 +211,15 @@ class TransactionHistoryViewController: UIViewController {
 
 extension TransactionHistoryViewController: UITableViewDataSource, UITableViewDelegate {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+
+        if(self.Filtercontent.count == 0){
```

Comment:
- brackets not needed in if condition
- space after `if` keyword and before opening brace `{`